app/js/controllers/actions/membership/membership-list-ctrl.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('MembershipListCtrl', function($scope, $rootScope, $stateParams, $state, Restangular, apiDescriptor, dataTransformer) {
+	var resourceName = $stateParams.resourceName;
+	var resourceId = $stateParams.id;
+	$scope.resourceName = resourceName;
+	apiDescriptor.then(function(apiDescription) {
+		$scope.rdesc = apiDescription.resource(resourceName);
+	});
+
+
+	var selectionMode = $stateParams.selectionMode;
+	if (!selectionMode || (selectionMode !== 'single ' && selectionMode !== 'multiple')) {
```

Comment:
not sure what the `selectionMode` stuff is/was being used for. If it's not important can you remove it